browser/brave_wallet/keyring_service_unittest.cc

```diff
@@ -118,6 +118,14 @@ class TestKeyringServiceObserver
   }
 
   void AccountsChanged() override { accounts_changed_fired_count_++; }
```

Comment:
I kept AccountsChanged since I did not implement AccountsRemoved in this change.